src/attack.c

```diff
@@ -890,8 +888,20 @@ apply_source_iface_preference (void)
 
       g_free (ipstr);
       g_free (ip6str);
-      return 0;
+      ret = 0;
```

Comment:
```suggestion
```
Not needed. Ret is already 0.

Reply:
Fixed with https://github.com/greenbone/openvas-scanner/pull/695/commits/e36f8545effc55b769d772c936d38bff1a520909